controllers/proxy/proxy_controller.go

```diff
@@ -0,0 +1,110 @@
+package proxy
+
+import (
+	"context"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	configv1 "github.com/openshift/api/config/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/openshift/splunk-forwarder-operator/api/v1alpha1"
+	"github.com/openshift/splunk-forwarder-operator/config"
+)
+
+var log = logf.Log.WithName("controller_proxy")
+
+type ProxyReconciler struct {
+	Client client.Client
+	Scheme *runtime.Scheme
+	Config *corev1.ConfigMap
+}
+
+func Reconcile(proxy *configv1.Proxy, c client.Client) (*corev1.ConfigMap, error) {
```

Comment:
Does this Reconcile function need to be exported? As far as I can tell it's only being called on line 96 in this file.